NodeJS/NodeMailer/README.md

```diff
@@ -0,0 +1,43 @@
+# Node Contact Form
+
+Simple Node.js/Express app using Nodemailer to send emails
+
+- Please add your own SMTP info for it to work
+
+### Version
+
+1.0.0
+
+## Install Dependencies
+
+```bash
+1. express
+2. body-parser
+3. express-handlebars
+4. nodemailer
+5. path
+```
+
+
+
+## Quick Start
+
+```bash
+1. git clone https://github.com/abhijeet007rocks8/Dev-Scripts.git
+2. cd NodeJs
+3. cd ModeMailer
+4. npm i
+5. nodemon app
+6. Goto browser and do localhost:5000
+```
+
+##video
+
+```bash
+https://user-images.githubusercontent.com/82238106/170574088-34c9111e-f69e-4be7-b4fb-526a95162e30.mp4
+```
```

Comment:
don't add this code block here for this video, as you can see in the preview that after adding the code block, the video is not accessible.
So remove these backticks that you have used for code block.